spec/models/post_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'Title must be present' do
+    post = Post.new
+    expect(post).to_not be_valid
+  end
+
+  it 'must be associated wit a user' do
+    post = Post.new(user: nil)
+    expect(post).to_not be_valid
+  end
+
+  it 'Title must not be too long' do
+    user = User.create(Name: 'John Doe', Bio: 'Software developer')
+    title = 'hello' * 100
+    post = Post.create(Title: title, user: user)
+    expect(post).to_not be_valid
+  end
```

Comment:
- [ ] I would recommend adding the test for the positive scenarios as well. Ex:


```ruby
describe 'post title validations' do
  it { should validate_presence_of(:title) }
end
```

Reply:
This issue has been addressed well. Nice job! :clap: :clap: